apps/remix-ide/src/app/files/git-handle.js

```diff
@@ -0,0 +1,27 @@
+import { WebsocketPlugin } from '@remixproject/engine-web'
+import * as packageJson from '../../../../../package.json'
+
+const profile = {
+  name: 'git',
+  displayName: 'Git',
+  url: 'ws://127.0.0.1:65521',
+  methods: ['command'],
```

Comment:
Why `command` while the scriptRunner method is `execute` ? I think it makes it difficult for developers to follow up with the naming.
I would recommend to define a kind "terminal" or whatever with the right interface.